variables.tf

```diff
@@ -108,6 +108,24 @@ variable "s3_logs_sqs_receive_wait_time_seconds" {
   default     = 10
 }
 
+variable "external_s3_logs_bucket_account" {
+  description = "This is the s3 logs bucket account where the s3 logs sqs should receive event notifications from."
+  type        = string
+  default     = ""
+}
+
+variable "external_s3_logs_bucket_account_region" {
+  description = "This is the environment account region where the s3 logs sqs should receive event notifications from."
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "external_prefix_s3_logs_bucket" {
+  description = "This is the s3 logs bucket prefix where the s3 logs sqs should receive event notifications from."
+  type        = string
+  default     = "federated-data-lake"
```

Comment:
Remove this default please. This would mean anybody who use this project will get this bucket added by default